.github/workflows/ci.yml

```diff
@@ -20,8 +20,20 @@ jobs:
       options: --privileged
```

Comment:
naming this job test-storge-user will be more consistent and clear.

Reply:
Ok, but then I need the project settings to change `tests-storage` from the required checks before merging for the new jobs.

Reply:
Of course we need to adapt the call calling this to do this change.